transfer.py

```diff
@@ -0,0 +1,184 @@
+    flight = message.flight
+
+    point = Point(measurement="HP", time=flight.timestamp.isoformat())
+
+    point.tags["centre"] = flight.centre
+    # Effectively an alias for "centre"
+    point.tags["center"] = flight.centre
+
+    if flight.arrival is not None:
+        _arrival(flight.arrival, point)
+
+    if flight.flightIdentification is not None:
+        _flight_identification(
+            flight.flightIdentification,
+            point)
+
+    point.tags["flightStatus"] = flight.flightStatus.fdpsFlightStatus
+    point.fields["gufi"] = flight.gufi.valueOf_
+
+    if flight.supplementalData is not None:
+        _supplemental_data(flight.supplementalData, point)
+
+    _coordination(flight.coordination, point)
+
+    # In accordance with the requirements doc, we're renaming this field
+    point.fields["ERAM_GUFI"] = flight.flightPlan.identifier
+
+    point.check_types()
+    return point
+
+
+def th_message(message):
+    """Converts a FIXM message from the TH source to an InfluxDB point."""
+    flight = message.flight
+
+    point = Point(measurement="TH", time=flight.timestamp.isoformat())
+
+    point.tags["centre"] = flight.centre
+    # Effectively an alias for "centre"
+    point.tags["center"] = flight.centre
+
+    if flight.arrival is not None:
+        _arrival(flight.arrival, point)
+
+    if flight.controllingUnit is not None:
+        point.fields["controllingUnit.unitIdentifier"] = \
+            flight.controllingUnit.unitIdentifier
+        point.fields["controllingUnit.sectorIdentifier"] = \
+            flight.controllingUnit.sectorIdentifier
+
+    if flight.departure is not None:
+        point.fields["departure.point"] = flight.departure.departurePoint
+        if flight.departure.runwayPositionAndTime is not None:
+            _runway_position_and_time(
+                obj=flight.departure.runwayPositionAndTime,
+                point=point,
+                name="departure")
+
+    if flight.enRoute is not None:
+        _en_route(flight.enRoute, point)
+
+    if flight.flightIdentification is not None:
+        _flight_identification(
+            flight.flightIdentification,
+            point)
+
+    point.tags["flightStatus"] = flight.flightStatus.fdpsFlightStatus
+    point.fields["gufi"] = flight.gufi.valueOf_
+    if flight.operator is not None:
+        point.tags["operator"] = \
+            flight.operator.operatingOrganization.organization.name
+
+    if flight.supplementalData is not None:
+        _supplemental_data(flight.supplementalData, point)
+
+    if flight.assignedAltitude is not None \
+            and flight.assignedAltitude.simple is not None:
+        point.fields["assignedAltitude"] = \
+            float(flight.assignedAltitude.simple.valueOf_)
+
+    # In accordance with the requirements doc, we're renaming this field
+    point.fields["ERAM_GUFI"] = flight.flightPlan.identifier
+
+    point.check_types()
+    return point
+
+
+def _arrival(obj, point):
```

Comment:
I really feel like you should have subclassed Point instead of having all these module-level functions.

At minimum they should be named such that it's clear they modify one of the input parameters (e.g. `_set_arrival`)